src/librustdoc/html/static/main.js

```diff
@@ -215,14 +215,14 @@
         } else if (ev.target.tagName === 'SPAN' && hasClass(ev.target.parentNode, 'line-numbers')) {
             var prev_id = 0;
 
-            function set_fragment(name) {
```

Comment:
it's not clear to me why ES5 wouldn't accept this, but not a big deal, seems fine to me